packages/bootstrap/scss/toolbar/_variables.scss

```diff
@@ -43,6 +43,19 @@ $kendo-toolbar-md-spacing: k-spacing(2) !default;
 /// @group toolbar
 $kendo-toolbar-lg-spacing: k-spacing(2.5) !default;
 
+/// The height of the Toolbar button separator.
+/// @group toolbar
+$kendo-toolbar-separator-height: null !default;
```

Comment:
I think we should either set this variable to `$kendo-button-md-calc-size` and reuse it for `$kendo-toolbar-md-separator-height` or remove it since currently it is not used anywhere.

> NB: Applies to all themes

Reply:
I'm using it in the layout:

```
// Overflow separator
.k-toolbar-button-separator {
    height: $kendo-toolbar-separator-height;
    margin-inline-start: auto;
}
```